composer.json

```diff
@@ -23,18 +27,25 @@
         "dflydev/dot-access-data": "^1.1.0 || ^2 || ^3",
         "grasmash/yaml-expander": "^3",
         "loophp/phposinfo": "^1.7.1",
-        "symfony/filesystem": "^6.2",
+        "symfony/filesystem": "^5.4 || ^6.2",
         "symfony/yaml": "^4.4 || ^5 || ^6"
     },
-    "conflict": {
-        "acquia/blt": "*"
-    },
     "require-dev": {
         "acquia/coding-standards": "^1.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
-        "ergebnis/composer-normalize": "~2.15.0",
+        "ergebnis/composer-normalize": "~2.21.0",
         "phpro/grumphp-shim": "^1.5"
     },
+    "conflict": {
+        "acquia/blt": "*"
```

Comment:
@deepakmishra2 Are you sure that this is correct?

Reply:
this is already there, the code is only normalized. As @vishalkhode1 mentioned until we completely remove dependency of blt from DRS we cannot remove this BLT conflict